ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SyslogIntegrationForm.tsx

```diff
@@ -53,6 +53,14 @@ export const validationSchema = yup.object().shape({
             useTls: yup.bool(),
             skipTlsVerify: yup.bool(),
         }),
+        maxMessageSize: yup
+            .number()
+            .required('Message size is required')
```

Comment:
I would flip the order of these, because otherwise `required` is redundant if it passed the number check.

```suggestion
            .required('Message size is required')
            .number()
```

Reply:
You're right; I guess I've been out of the game too long.